src/dash-table/derived/table/tooltip.ts

```diff
@@ -24,34 +31,55 @@ function getSelectedTooltip(
         return undefined;
     }
 
-    const {id, row} = currentTooltip;
+    const {header, id, row} = currentTooltip;
 
     if (id === undefined || row === undefined) {
         return undefined;
     }
+    const conditionalTooltips = header
+        ? []
+        : R.filter(tt => {
+              return (
+                  !tt.if ||
+                  (ifColumnId(tt.if, id) &&
+                      ifRowIndex(tt.if, row) &&
+                      ifFilter(
+                          tt.if,
+                          virtualized.data[row - virtualized.offset.rows]
+                      ))
+              );
+          }, tooltip_conditional);
+
+    if (conditionalTooltips.length) {
+        return conditionalTooltips.slice(-1)[0];
+    }
```

Comment:
nonblocking, but perhaps https://ramdajs.com/docs/#findLast would be cleaner here?

Reply:
Sure https://github.com/plotly/dash-table/pull/831/commits/8eec8201d28a2560fd01e078e178ddfa88a1284b